test/probe/http-client.test.js

```diff
@@ -819,4 +820,44 @@ describe("probe http-client", function() {
             assert.strictEqual(this.metrics, undefined);
         });
     });
+
+    describe("chunked encoding", function() {
+        let realServer;
+        const port = 8000; //server.getHttpPort() + 1;
+        const expectedText = 'first line\nsecond line\nthird line\n';
+
+        before("create server", function () {
+            realServer = http.createServer((req, res) => {
```

Comment:
We already use `nock` and `mock-http-server` for mocking http requests, so I would not want to add yet another way if not needed (see above).